examples/pytorch/image-classification/run_image_classification.py

```diff
@@ -377,13 +377,18 @@ def val_transforms(example_batch):
             checkpoint = last_checkpoint
         train_result = trainer.train(resume_from_checkpoint=checkpoint)
         trainer.save_model()
-        trainer.log_metrics("train", train_result.metrics)
-        trainer.save_metrics("train", train_result.metrics)
+        metrics = train_result.metrics
+        metrics['throughput'] = metrics['train_samples_per_second']
+        metrics['loss']= metrics['train_loss']
+        metrics['lr'] = training_args.learning_rate
```

Comment:
Learning rate is a training hyper-parameter, not a metrics. I believe this is not quite the right place to log LR. Can you find a more appropriate place to log this info?